tsconfig.aot.json

```diff
@@ -10,24 +10,28 @@
     "noImplicitAny": false,
     "module": "es2015",
     "moduleResolution": "node",
-    "rootDir": "./src",
-    "outDir": "./src",
     "sourceMap": true,
     "inlineSources": true,
     "skipLibCheck": true,
     "lib": [
       "es2015",
       "dom"
     ],
+    "typeRoots" : [
+      "./src/typings",
+      "./node_modules/@types"
+    ],
     "types": [
-      "node"
+      "node",
```

Comment:
maybe remove "types" .
from the doc: If types is specified, only packages listed will be included